### PR TITLE
Hot fix #435 order-dependence breaking clean build on Linux

### LINKED ISSUE #435: [dev] Getting "Cannot read property 'prototype' of undefined" after clean build
```make``` ran without any errors. Running the server and going to unit.html page I get this error. 

Error is at line: ```_.selectTowards = MathCommand.prototype.selectTowards;```

Anyone else getting this?

### PR DESCRIPTION
**Pull request by laughinghan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,8 +20,12 @@ BASE_SOURCES = \
...
 SOURCES_FULL = \
   $(BASE_SOURCES) \
-  $(SRC_DIR)/commands/*.js \
-  $(SRC_DIR)/commands/*/*.js
+  $(SRC_DIR)/commands/math.js \
+  $(SRC_DIR)/commands/text.js \
+  $(SRC_DIR)/commands/math/*.js
+# text.js currently depends on math.js #435 :(
+# $(SRC_DIR)/commands/*.js \
+# $(SRC_DIR)/commands/*/*.js
```
Comment (stufflebear) [inline Makefile]: dead comments

Comment (laughinghan) [inline Makefile]: I'm keeping them around as a FIXME

Comment (stufflebear) [inline Makefile]: do you think that's clear to other maintainers?

Comment (laughinghan) [inline Makefile]: Good point, is the update comment any better?

Comment (stufflebear) [inline Makefile]: yeah, that's a lot clearer

**Comment by stufflebear** [comment]
+1

**Pull request merged**